### DISCUSSION TITLE
addDoubletScores Error - outputDirectory cannot have a space in the full path!

### DISCUSSION THREAD

**Discussion by janekim319** [GreenleafLab/ArchR#918] [Questions / Documentation] [answered]
I'm simply following the tutorial, using the same data. I successfully ran createArrowFiles() and the variable "ArrowFiles" prints correctly: [1] "scATAC_BMMC_R1.arrow"      "scATAC_CD34_BMMC_R1.arrow" "scATAC_PBMC_R1.arrow"  . However, when I run 

doubScores <- addDoubletScores(
    input = ArrowFiles,
    k = 10, #Refers to how many cells near a "pseudo-doublet" to count.
    knnMethod = "UMAP", #Refers to the embedding to use for nearest neighbor search with doublet projection.
    LSIMethod = 1
)

I get an error:
Error in ArchRProject(ArrowFiles = ArrowFile, outputDirectory = tmpDir, : outputDirectory cannot have a space in the full path! Full path : /Users/jeongeunkim/Desktop/HARVARD/Dana Farber Research/tmp/tmp-35b91f1e29cb-Date-2021-07-21_Time-18-05-48

When I pass in an argument "outDir = getOutputDirectory(input)", I get an error: Error in getOutputDirectory(input) : object 'input' not found

My working directory is the directory that contains the arrow files (/Users/jeongeunkim/Desktop/HARVARD/Dana Farber Research). There are other non-arrow files also in the same directory.

**Comment by janekim319** [comment, marked as answer]
Ah I realized that the error message means that the directory name cannot have any spaces in it. I figured my directory name of "Dana Farber Research" was the issue! You can ignore my question!

**Reply by rcorces** [reply]
sure. in the future, errors go to Issues, usage questions and feature requests go here in Discussions
